playbook/app/pb_kits/playbook/pb_button/_button_mixins.scss

```diff
@@ -5,6 +5,7 @@
 @import "../tokens/spacing";
 @import "../tokens/transition";
 @import "../tokens/typography";
+@import "../tokens/accessibility";
```

Comment:
This is not really a token. Whats our strategy for re-useable mixins? @jasperfurniss

Reply:
Yeah I was not expecting this to be a token. Only a mixin. Because we will indeed be using similar patterns across multiple kits.

Reply:
Do we have a mixins folder? Or should one be created? We also have titles.scss in the tokens folder which is a mixin folder. So we could move both to a new mixin folder?

Reply:
Ideally, the mixins would live close to their corresponding kit. For example, button mixin should live close to the button component. 

In our case, however, we are attempting to create a mixin that is not kit specific and will span across multiple kits. 

I think making a new mixin folder for now is better structure for us, and clears up confusion.  
So, yes.